lib/src/better_feedback.dart

```diff
@@ -2,17 +2,29 @@ import 'package:feedback/feedback.dart';
 import 'package:feedback/src/feedback_controller.dart';
 import 'package:feedback/src/feedback_functions.dart';
 import 'package:feedback/src/feedback_widget.dart';
+import 'package:feedback/src/string_feedback.dart';
 import 'package:feedback/src/theme/feedback_theme.dart';
 import 'package:feedback/src/utilities/feedback_app.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:feedback/src/l18n/translation.dart';
 
-/// This widget should be at the top of your widget tree.
+/// The function to be called when the user submits their feedback.
+typedef OnSubmit = void Function(Object feedback);
+
+/// A function that returns a Widget that prompts the user for feedback and
+/// calls [OnSubmit] when the user wants to submit their feedback.
+typedef GetFeedback = Widget Function(OnSubmit);
```

Comment:
Yeah I thought about that and went without it as users can just use a `builder` if they want access to a `BuildContext`-it'll just pollute the function arguments otherwise. That said, it's pretty conventional in Flutter to pass a `BuildContext` in this kind of situation.

How about I rename `GetFeedback` to `FeedbackBuilder` and add a `BuildContext` argument?